test/src/module/db/dbTest.c

```diff
@@ -143,6 +143,7 @@ testRun(void)
         strLstAddZ(argList, "--stanza=test1");
         strLstAddZ(argList, "--repo1-retention-full=1");
         strLstAddZ(argList, "--pg1-path=/pg1");
+        strLstAddZ(argList, "--pg1-database=backupdb");
```

Comment:
I would have liked to see a failure test but that would require a live postgres. I'll just check it manually.

Reply:
Right, and I don't think it is worth the extra time spent.